portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,69 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Gets comments for the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getComments() {
+  const commElem = document.getElementById('comments-container');
+  commElem.innerHTML = "";
+  var number = document.getElementById("number").value;
+  fetch("/data?number=" + number).then(response => response.json()).then((comments) => {
+    const commElem = document.getElementById('comments-container');
+     comments.forEach(c => {
+        var textDiv = document.createElement("div");
+        textDiv.innerHTML = "<div class='comment-text'>" + c.text + "</div>";
+        textDiv.classList.add("box");
+        commElem.appendChild(textDiv);
+        var infoDiv = document.createElement("div");
+        infoDiv.innerHTML = "<span class='comment-name'>" + c.name + " " + "</span>" + 
+                      "<span class='comment-time'>" + getTimeStamp(c) + "</span>";
+        infoDiv.classList.add("comment-info");
+        commElem.appendChild(infoDiv);
+    });
+    if (comments.length > 0) {
+      buttonDiv = document.getElementById("delete");
+      buttonDiv.innerHTML = "";
+      var button = document.createElement("button");
+      button.onclick = function() { deleteComments(); }; 
+      if (comments.length === 1) {
+        button.innerText = "Delete Only Comment";
+      }
+      else {
+        button.innerText = "Delete All " + comments.length + " Comments";
+      }
+      buttonDiv.appendChild(button);
+    }
+  }); 
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function deleteComments() {
+  fetch("delete-data", {method: "POST"});
+  window.location.reload();
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getTimeStamp(comment) {
+  var time = ((new Date()).getTime() - comment.timestamp)/1000; 
+  var seconds = (time).toFixed(0);
```

Comment:
Prefer "const" over "var" if you're not going to reassign these.

Reply:
Fixed!